src/index.html

```diff
@@ -576,6 +576,7 @@ <h6>Orders</h6>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49"
     crossorigin="anonymous"></script>
   <script src="./scripts/bootstrap.min.js"></script>
+  <script src="./scripts/stars.js"></script>
```

Comment:
Dodawanie nowego pliku do ładowania w html, nei jest dobrą drogą. Zawsze lepiej miec jeden wiekszy plik niz kilka małych. Chcesz trzymać skrypt z gwiazdkami w innym pliku? Okej, ale w takim razie wyapdałoby importować to po prostu w app.js, tak, żeby na końcu miec po dolaczony tylko ten plik. Albo po prostu od razu wklej ten kod do app.js zamiast robic go osobno.

Reply:
Wyciągnięto kod do App.js.